From 525f66e63b29c4436cf0743e562be28c00de0fc6 Mon Sep 17 00:00:00 2001
From: He Junyan <junyan.he@intel.com>
Date: Wed, 26 Jun 2024 22:19:52 +0800
Subject: [PATCH 74/98] vadeinterlace: Do not append ANY caps into pad template

Just like the vapostproc, we delete the ANY caps in pad template to
avoid unexpected negotiation.

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/6698>
---
 sys/va/gstvadeinterlace.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/sys/va/gstvadeinterlace.c b/sys/va/gstvadeinterlace.c
index 5337c32862bf..f7bef8a0cb0f 100644
--- a/sys/va/gstvadeinterlace.c
+++ b/sys/va/gstvadeinterlace.c
@@ -724,12 +724,6 @@ gst_va_deinterlace_class_init (gpointer g_class, gpointer class_data)
 
   if (gst_va_filter_open (filter)) {
     src_caps = gst_va_filter_get_caps (filter);
-    /* adds any to enable passthrough */
-    {
-      GstCaps *any_caps = gst_caps_new_empty_simple ("video/x-raw");
-      gst_caps_set_features_simple (any_caps, gst_caps_features_new_any ());
-      src_caps = gst_caps_merge (src_caps, any_caps);
-    }
   } else {
     src_caps = gst_caps_from_string (caps_str);
   }
-- 
2.43.0

